### PR TITLE
Test/time actions

### PR DESCRIPTION
**Pull request by guhmerces**

### CODE REVIEW

**Comment by guhmerces** [comment]
Adding info:

image

**Comment by rjrodger** [comment]
Did you have to create test or developer newrelic account to validate this? does it generate reports/charts?

**Review by rjrodger — CHANGES_REQUESTED** [review]

File: create_config_file.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+
+process.on('uncaughtException', (err) => {
+    console.log(err)
+    process.exit(1);
+})
+
+createNewRelicConfigFile()
+
+function createNewRelicConfigFile(callback) {
```
[the remark this answers is not in the document]
Comment (guhmerces) [reply create_config_file.js:10]: Absolutely. I tried to automate the process but there isn't much to do. Like you mentioned, is a external config that's done manually.

File: package.json [hunk 2]
```diff
@@ -53,6 +54,7 @@
     "seneca-promisify": ">=2"
   },
   "dependencies": {
+    "newrelic": "^8.10.0",
     "trello": "^0.11.0"
```
[the remark this answers is not in the document]
Comment (guhmerces) [reply package.json:58]: yes, done

File: test/basic.js [hunk 3]
```diff
@@ -1,74 +1,105 @@
 
 
 const Seneca = require('seneca')
+const newrelic = require('newrelic')
+
+function newSegment(actionCb) {
```
[the remark this answers is not in the document]
Comment (guhmerces) [reply test/basic.js:6]: This one is out of date. For the moment, I calling the newrelic API methods directly

File: test/basic.js [hunk 4]
```diff
@@ -1,74 +1,105 @@
...
 
-      await sleep(100)
 
-      // NewRelic end?
-      actionEnd(meta)
-      
-      return {x:msg.x}
-    })
+// Basic message 
+    .message('a:1', startTransaction(
```
[the remark this answers is not in the document]
Comment (guhmerces) [reply test/basic.js]: out of date
discussion extended in slack 

thanks

**Comment by guhmerces** [comment]
I've created a new PR. Perhaps this pr should be closed ?

**Pull request closed without merging**